bulk-model-sync-mps/src/main/kotlin/org/modelix/mps/model/sync/bulk/MPSBulkSynchronizer.kt

```diff
@@ -78,8 +80,13 @@ object MPSBulkSynchronizer {
         access.runWriteInEDT {
             access.executeCommand {
                 val repoAsNode = MPSRepositoryAsNode(repository)
+                fun moduleFilter(node: INode): Boolean {
```

Comment:
Interesting. When we use as the MPSRepositoryAsNode root node for the import, but filter the modules we want to import. It would try to delete modules that are not imported.

For that you introduced the filter. Should we add this as a comment?

Reply:
It was already part of the commit message, but I added a comment for better visibility.